Dev (#47)
42-AI/bootcamp_data-engineering: day01/ex09/ex09.md

> # Exercise 09 - Aggregation in Aggregation

Reviewer: What about `Aggregation-ception` or something like this?

|                         |                    |
| -----------------------:| ------------------ |

> |   Turnin directory :    |  ex09              |

Reviewer: ```suggestion
|   Turn-in directory:    |  ex09              |

```

> |   Files to turn in :    |  ex09-rendu.txt    |

Reviewer: ```suggestion
|   Files to turn in:     |  ex09-rendu.txt    |

```

> |   Forbidden function :  |  None              |

Reviewer: ```suggestion
|   Forbidden functions:  |  None              |

```

> |   Remarks :             |                    |

Reviewer: ```suggestion
|   Remarks:              |                    |

```

> We would like to know what is the most represented game "Genre" (top 10) and for each of those "genre" the repartition of the "Average User Rating" with a bucket interval of one (ie: for each Genre the number of game with an Avg User Rating of 1, of 2, 3, 4, and 5).

Reviewer: ```suggestion
We would like to know what is the most represented game "Genre" (top 10) and for each of those "genre" the repartition of the "Average User Rating" with a bucket interval of one (i.e., for each Genre the number of game with an Avg User Rating of 1, of 2, 3, 4, and 5).

```
